libs/design/modal/examples/src/basic-modal/basic-modal.component.html

```diff
@@ -1 +1 @@
-<button daff-button (click)="showModal()">Open Modal</button>
+<button daff-button (click)="showModal()" aria-haspopup="dialog">Open Modal</button>
```

Comment:
Remove `aria-haspopup="dialog"`. Out of scope.

Reply:
Moved to #4167